core/validation/pre_transformation_validate.py

```diff
@@ -123,7 +123,7 @@ def check_values(
     failures = []
 
     for sheet, column, row, expected_values, type in failure_list:
-        expected_value = workbook[sheet].iloc[column][row]
+        expected_value = str(workbook[sheet].iloc[column][row]).strip()
```

Comment:
I think this should actually be ```entered_value``` as we use the column and row to get the actual value. Otherwise a bit confusing as we're supposed to do entered vs expected.